.github/workflows/main.yml

```
@@ -3,12 +3,30 @@ name: Exercism Erlang Track verification
 on: [push, pull_request]
 
 jobs:
+  create_otp_matrix:
+    name: Generate a list of last OTP versions
+    runs-on: ubuntu-latest
+    outputs:
+      otps: ${{ steps.versions.outputs.versions }}
+    steps:
+    - name: Clone the repository
+      uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
+    - name: Get latest OTP versions
+      id: versions
+      env: 
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        all_versions=$(gh api graphql -f query='query { repository(owner: "erlang", name: "otp") { releases(last: 100) { nodes { tagName } } } }' --jq '.data.repository.releases.nodes[].tagName | select(. | contains("rc") | not) | .[4:8]' | sort -u)
```

In case they ever reach a minor version of 10, numerical sorting would be good instead of alphabetical:
```suggestion
        all_versions=$(gh api graphql -f query='query { repository(owner: "erlang", name: "otp") { releases(last: 100) { nodes { tagName } } } }' --jq '.data.repository.releases.nodes[].tagName | select(. | contains("rc") | not) | .[4:8]' | sort -u -n)
```

---

Oh, you are right. And I will incorporate your suggestion, even though in this case the minor isn't expected to ever exceed 3, as the release schedule is one major per year, and a minor every 3 months.

Also the slice I use (`[4.8]`) will only ever allow for two digit minors if majors become single digit, or the tagging schema changes fundamentally, which would get me into a different kind of trouble :D

---

And I just realize, the sorting isn't even important here, all I need is getting doubles out of the list.